src/engine/helper.rs

```diff
@@ -0,0 +1,275 @@
+use serde_json::Value;
```

Comment:
the theme seems to be evaluation in this file - would it make sense to rename the module to reflect that ? 
Or maybe there are more topics hidden in there and it could be split ? Not sure here ?

Reply:
Yeah, name things. The problem I see with `eval.rs` name is that the `mod.rs` contains `Engine` and there's `eval` and `eval_as_bool`. This one is kind of high level wrapper around all these things. Dunno.